General/Baselines.md

```diff
@@ -111,13 +111,13 @@ baseline: spec
 				package: 'MyProject-Examples' ]
 ```
 
-> Note: Packages are the most atomic entities managed by the baseline. It is not possible to declare entities at the package tag granularity.
+> Note: Packages are the most atomic entities managed by the baseline. It is not possible to declare entities at the package-tag granularity.
 
 Defining packages is not enough to load them, because some of them might depend on other packages/projects. For example, `MyProject-Tests` needs to be loaded after `MyProject`.
 
 To manage dependencies that are external to a project, see section *[Define external dependencies](#define-external-dependencies)*.
 
-For dependencies between the packages of your project, you can use the message `#package:with:` to give more informations to the spec.
```

Comment:
Maybe at some point I'll stop to do this mistake again and again and again :(

Reply:
I had a similar problem with example/exemple in French